### DISCUSSION TITLE
Are you open for design contributions?

### DISCUSSION THREAD

**Discussion by fivenp** [pkgxdev/pkgx#262] [Ideas]
Hey there! 

I'm loving tea in my daily use but thought it might benefit here and there potentially from some design driven input (or having in general designers who dedicate some time to the project). 

Therefor I've built https://contribute.design which helps designers and open source software to understand each other better. Basically I'm trying to help project get the guidelines for designers set up and create some visibility across designers for the project. 

My question now is – would you be open to participate in this initiative and can I potentially support you in any way here? :)

**Comment by mxcl** [comment]
@tsmitty11 is our designer so he can chime in.

**Reply by tsmitty11** [reply]
@fivenp love the idea, def open to design contributions. I'll take a look at your site and see how we can work together!

**Reply by fivenp** [reply]
amazing @tsmitty11! If I can do anything to help you getting started feel free to shoot! :)

**Reply by tsmitty11** [reply]
@fivenp thanks! This is such a cool idea. We'll get our .design folder and README sorted this week.

**Reply by tsmitty11** [reply]
Will let you know if we have any questions during that process!
